canvas_modules/common-canvas/__tests__/test_resources/paramDefs/checkboxset_paramDef.json

```diff
@@ -263,6 +299,31 @@
           "default": "checkboxset that is hidden"
         }
       },
+      {
+        "parameter_ref": "filter",
+        "label": {
+          "default": "Filter 'checkboxset Filtered'"
+        }
+      },
+      {
+        "parameter_ref": "filter2",
+        "label": {
+          "default": "Filter 'fruit in table'"
+        },
+        "description": {
+          "default": "Fruit to have values 'orange' and 'pear' when new row is added to table"
+        }
+      },
+      {
+        "parameter_ref": "checkboxset_filtered",
+        "label": {
+          "default": "checkboxset Filtered"
+        },
+        "resource_key": "checkboxset_filtered",
+        "description": {
+          "default": "checkboxset filtered to have values 'red' and 'blue'"
```

Comment:
This should be `'apple' and 'pear'`
<img width="319" alt="image" src="https://user-images.githubusercontent.com/1628100/79781505-6f58ab80-82f2-11ea-983f-2dcf2e308433.png">

Reply:
good catch.  I'll get this fixed.